src/renault_api/kamereon/models.py

```diff
@@ -64,6 +64,11 @@
     "XBG1VE": {  # DACIA SPRING
         "control-charge-via-kcm": True,
     },
+    "XCB1VE": {  # MEGANE E-TECH
+        "reports-in-watts": True,
+        "reports-charge-session-durations-in-minutes": True,
+        "support-endpoint-lock-status": False,
```

Comment:
I have no way to check, but this seems weird.
Are you sure that your vehicle doesn't report in kilowatts, and doesn't report charge session durations in minutes?

Reply:
@epenet The car definitelly reports in `kilowatts` based on the data I have shared in the description. But I thought the unit for the instant charging power was `amps` (that's what Home Assistant integration thinks at least, it changes to `kW` if the car is marked with `"reports-in-watts": True`). What is the base unit `renault-api` expects here?

As for the charging session, it reports in minutes indeed. Isn't it what `"reports-charge-session-durations-in-minutes": True` means?

Reply:
For 1st generation vehicles, the data was reported in "watts" (and it gets divided in home assistant by 1000 to report in kilowatts)

For 2nd generation vehicles, nobody really knew what it was (kilowatts? or amps?) so I think you should open a PR on Home Assistant to set it back to kilowatts not amps.

Reply:
For your vehicle, I'm pretty sure the default should be left for the first two items.
For lock status I trust your judgment.
```suggestion
        "support-endpoint-lock-status": False,
```

Reply:
Sure @epenet, I'll update. And thank you for the PR on Home Assistant!

Reply:
FYI, the lock-status endpoint is definitely not supported with the Mégane, that returns a 403 when I'm trying to get the data.